src/platform/forms-system/test/js/components/FormNav.unit.spec.jsx

```diff
@@ -120,7 +120,8 @@ describe('Schemaform FormNav', () => {
     );
   });
 
-  it('should display/return correct chapter title when title-function uses onReviewPage parameter', () => {
+  // skipping due to flakiness in CI
+  it.skip('should display/return correct chapter title when title-function uses onReviewPage parameter', () => {
```

Comment:
I think I reported the previous tests on line 108 as the flaky test.

Reply:
@Mottie , corrected.  Sorry about that!